### PR TITLE
Added a quick bootstrap script to the package.json file.

### PR DESCRIPTION
**Pull request by MorganCabral**
To quickly bootstrap a development environment, run `npm run bootstrap`.
This will install bower, jison and gulp globally for you.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "test": "gulp ci"
+    "test": "gulp ci",
+    "bootstrap": "npm install -g bower gulp jison"
```
Comment (RobertWarrenGilmore) [inline package.json]: We need a readme with a set of terminal steps from "nothing" state to running the app.

Comment (RobertWarrenGilmore) [inline package.json]: "nothing" means I have no Git, I have no NPM, I have no Gulp. All I have is a directory and a terminal. Also, this should cover Windows and Linux.

Comment (RobertWarrenGilmore) [inline package.json]: Well, maybe not that "nothing". Maybe I've just cloned from Git.

File: README.md [hunk 2]
```diff
@@ -25,10 +25,23 @@ bower install goll-e
...
 npm install
-bower install
+
+# Run the test suite.
 gulp ci
+
+# Install dependencies for the example implementation.
+cd examples
+npm install && bower install
+npm start
```
Comment (RobertWarrenGilmore) [inline README.md]: Nice. I like.

**Comment by RobertWarrenGilmore** [comment]
I approve.

**Comment by jboolean** [comment]
Cool.

**Approve**

**Pull request merged**